docs/labs/lab08-resource-apis/topology_tag_assignment.py

```diff
@@ -62,13 +80,21 @@
 request = 'REQUEST_START_BUILD'
 request_id = 'b1'
 description='testing cvprac build'
-clnt.api.workspace_config(workspace_id=workspace_id,display_name=display_name,description=description,request=request,request_id=request_id)
+build_ws = clnt.api.workspace_config(workspace_id=workspace_id, display_name=display_name, 
```

Comment:
Why add assignment of this build_ws variable if it is never used?

Reply:
hmm you're right, let me change that back